### DISCUSSION TITLE
[Feature Request] Sensor for: Power current [W]

### DISCUSSION THREAD

**Discussion by PV-Joe** [davidrapan/ha-solarman#59] [Ideas]
The **“Power current”** in Watts (total house consumption) should be:

**Load Power + Power losses**

Am I right?
If yes, would you like to add this calculated sensor permanently in your definition.yaml file?

**Comment by davidrapan** [comment]
That is really not a good idea cause even now I have to artificially constraint that Power losses sensor so it doesn't go negative. So in short Power losses is sensor which is really only good for having some idea about it but it's not good for any calculations at all.

**Reply by PV-Joe** [reply]
What a shame. It would have been so easy. Didn‘t know that you can‘t do further calculations with it.

**Comment by davidrapan** [comment]
I'm proly only using it here (above the load power):
![power_flow](https://github.com/user-attachments/assets/0a9e166f-a6f6-4bc3-98c4-215a1bdc6b21)
